### PR TITLE
Adding the Sudoku Project

### PR DESCRIPTION
**Pull request by soma2000-lang**
## Related Issue
  - The project is done using html ,javascript and css using the concept of backtracking

#1899

### Describe the changes you've made
Added all the files of the Sudoku Solver

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.

### Screenshots
WhatsApp Image 2021-07-18 at 02 19 06 (1)

### CODE REVIEW

**Comment by soma2000-lang** [comment]
@Rizwan-S 
Please look into this pull request

**Review by Rizwan-S — CHANGES_REQUESTED** [review]
Please follow the folder structure of the project. Put the html file in public folder and js, css files in relevant folders of assets folder. Also, please remove README file and also update this project's script.js file.

**Comment by soma2000-lang** [comment]
@Rizwan-S 
What changes shoud be done in script.js files

**Comment by Rizwan-S** [comment]
You need to add your project in the array.

**Comment by soma2000-lang** [comment]
@Rizwan-S  made all thr changes ,Please look into it

**Review by Rizwan-S — CHANGES_REQUESTED** [review]
Please rename 'Sudoku Solver.html' to 'Sudoku_Solver.html' & 'Sudoku Solver.jpeg' to 'Sudoku_Solver.jpeg'

**Comment by soma2000-lang** [comment]
@Rizwan-S  done

**Comment by Rizwan-S** [comment]
You need to rename the js and css file also. My bad, I didn't notice it first :(

File: public/Sudoku_Solver.html [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="style.css">
+    <script defer src="script.js"></script>
```
Comment (Rizwan-S) [inline public/Sudoku_Solver.html:11]: Use '../assets/js/Sudoku_Solver.js' as script src.

File: public/Sudoku_Solver.html [hunk 2]
```diff
@@ -0,0 +1,330 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="style.css">
```
Comment (Rizwan-S) [inline public/Sudoku_Solver.html:10]: Use '../assets/css/Sudoku_Solver.css' as href

**Review by Rizwan-S — CHANGES_REQUESTED** [review]

**Comment by soma2000-lang** [comment]
@Rizwan-S  Sir made the modifications.

**Comment by Rizwan-S** [comment]
Dude, this is wrong. Not everyone will place your files at that specific location. Please use what I mentioned.

**Comment by soma2000-lang** [comment]
@Rizwan-S  Sorry for my  mistake
Check now

**Review by Rizwan-S — APPROVED** [review]
Great Work!!!

**Comment by soma2000-lang** [comment]
Thank you sir

**Pull request merged**